### PR TITLE
Imprv/ndb delete method

### PR DESCRIPTION
**Pull request by Abhishekism9450**

### CODE REVIEW

File: nutanix/resource_nutanix_ndb_clone.go [hunk 1]
```diff
@@ -247,6 +253,42 @@ func resourceNutanixNDBClone() *schema.Resource {
 			},
 
 			"actionarguments": actionArgumentsSchema(),
+			// delete values
```
Comment (siddharth-kulshrestha) [inline nutanix/resource_nutanix_ndb_clone.go:256]: What is the significance of delete values comment? 
Please explain.

Comment (siddharth-kulshrestha) [inline nutanix/resource_nutanix_ndb_clone.go]: Please give bit more elaborative comments.

File: nutanix/resource_nutanix_ndb_clone.go [hunk 2]
```diff
@@ -608,7 +650,7 @@ func resourceNutanixNDBCloneUpdate(ctx context.Context, d *schema.ResourceData,
 	}
 
 	if res != nil {
-		log.Printf("NDB clone with %s id update successfully", d.Id())
+		log.Printf("NDB clone with %s id is update successfully", d.Id())
```
Comment (siddharth-kulshrestha) [inline nutanix/resource_nutanix_ndb_clone.go]:
```suggestion
		log.Printf("NDB clone with %s id is updated successfully", d.Id())
```

**Review by siddharth-kulshrestha — APPROVED** [review]

File: nutanix/resource_nutanix_ndb_clone.go (hunk 1 above)
Comment (Abhishekism9450) [reply nutanix/resource_nutanix_ndb_clone.go]: sure.

File: nutanix/resource_nutanix_ndb_clone.go (hunk 2 above)
Comment (Abhishekism9450) [reply nutanix/resource_nutanix_ndb_clone.go]: Noted.

**Pull request merged**